mysql-plugin/src/e2e-test/resources/pluginParameters.properties

```diff
@@ -0,0 +1,12 @@
+driver=mysql
+host=34.105.111.83
+username=dummy
+password=dummy
+database=sakila
+port=3306
+selectQuery=select * from
```

Comment:
nit: `selectQuery=select * from ${table}` looks more readable.
Can replace `${table}` with `sourceTable` in test setup hook.